requirements.txt

```diff
@@ -2,7 +2,7 @@ asgiref==3.2.10
 Django==3.1.2
 djangorestframework==3.12.1
 django-csp==3.7
-pkg-resources==0.0.0
```

Comment:
👍 I had to do this locally as well

Reply:
I vaguely recall there is/was a pip bug where `pip freeze` produced this.  And I did create requirements.txt by running `pip freeze`.